admin-dev/themes/default/template/controllers/categories/helpers/form/form.tpl

```diff
@@ -61,7 +61,4 @@
 {/block}
 {block name="input_row"}
 	{$smarty.block.parent}
-	{if ($input.name == 'thumb')}
-	{$displayBackOfficeCategory}
```

Comment:
Seems like you added back the Hook::exec() call, but how is the result displayed?